ee/localserver/krypto-ec-middleware.go

```diff
@@ -23,7 +23,8 @@ const (
 )
 
 type v2CmdRequestType struct {
-	Path string
+	Path          string
+	UrlParameters map[string]string
```

Comment:
I'm kinda willing to try, but I'm still really leery of max size stuff.

Reply:
https://github.com/kolide/launcher/pull/1030#discussion_r1107406286 =)

Reply:
Yeah, I lost track of the other comment. GitHub 🤷